tools/cfn2ts/lib/codegen.ts

```diff
@@ -555,7 +572,13 @@ export default class CodeGenerator {
       Object.keys(propTypeSpec.Properties).forEach(propName => {
         const propSpec = propTypeSpec.Properties[propName];
         const additionalDocs = quoteCode(`${typeName.fqn}.${propName}`);
-        const newName = this.emitProperty(resourceContext, propName, propSpec, additionalDocs);
+        const newName = this.emitProperty({
```

Comment:
this highlights the coupling I'm talking about in my larger comment where I'm seeking feedback.